### ISSUE TITLE
multipassd crashes when deleting a VM

### ISSUE THREAD

**Issue by OlegSmelov** [canonical/multipass#1266]
I can't delete a stopped VM:

```
$ multipass list              
Name                    State             IPv4             Image
quiet-burbot            Stopped           --               Ubuntu 18.04 LTS
$ multipass delete --all
delete failed: cannot connect to the multipass socket
Please ensure multipassd is running and '/var/snap/multipass/common/multipass_socket' is accessible
```

dmesg right after that:

```
[ 1669.814708] multipassd[32262]: segfault at 0 ip 00005590fa3efa14 sp 00007ffede3a4650 error 4 in multipassd[5590fa196000+736000]
[ 1669.814714] Code: 44 24 48 31 c0 8b 47 18 83 f8 07 0f 84 95 00 00 00 8d 50 fc 48 8b bf e8 01 00 00 83 fa 01 76 5e 83 f8 08 74 59 83 f8 02 74 44 <48> 8b 07 ff 90 90 00 00 00 48 8b bb e8 01 00 00 be 30 75 00 00 48
```

```
$ lsb_release -d
Description:	Ubuntu 18.04.3 LTS
$ snap info multipass | grep installed
installed:   1.0.1                           (1555) 185MB classic
```

I've also tried switching to the edge channel, but the problem persisted.

**Comment by DeepDarkOdyssey** [comment]
same issue here

**Comment by brettkoonce** [comment]
> delete failed: cannot connect to the multipass socket
> Please ensure multipassd is running and '/var/snap/multipass/common/multipass_socket' is accessible

**Comment by ricab** [comment]
Thanks for reporting, I confirm I can reproduce this. Looking into it.

**Comment by ricab** [comment]
Hi all! Could you confirm this fixes it for you?

`snap refresh multipass --channel edge/pr1278`

**Comment by OlegSmelov** [comment]
Works for me :+1:

**Comment by brettkoonce** [comment]
WFM, 🚀🏆🥳

**Comment by Saviq** [comment]
The fix for this is now in the [1.0.2 release](/canonical/multipass/releases/tag/v1.0.2), released in the `stable` snap channel.
